client/trino-client/src/main/java/io/trino/client/FixJsonDataUtils.java

```diff
@@ -56,28 +56,29 @@ final class FixJsonDataUtils
 {
     private FixJsonDataUtils() {}
 
-    public static Iterable<List<Object>> fixData(List<Column> columns, List<List<Object>> data)
+    public static Iterable<List<Object>> fixData(List<Column> columns, List<List<Object>> rows)
     {
-        if (data == null) {
+        if (rows == null) {
```

Comment:
you are changing this code multiple times in different commits. Hard to read. Please extract meaningless changes (renames etc) as preparatory commits and make one commit which changes logic here.